doc/stacker_yaml.md

```diff
@@ -84,6 +84,31 @@ will not be reflected.
 
 Will grab /path/to/file from the previously built layer `$name`.
 
+#### `import hash`
+
+The `import` directive also supports specifying the hash(sha256sum) of import source,
+for all the three forms presented above, for example:
+```
+import:
+  - path: config.json
+    hash: f55af805b012017bc....
+  - path: http://example.com/foo.tar.gz
+    hash: b458dfd63e7883a64....
+  - path: stacker://$name/path/to/file
+    hash: f805b012017bc769a....
+```
+Before copying the file it will check if the requested hash matches the actual one.
+This new import mode can be combined with the old one, for example:
+```
+import:
+  - path: "config.json
+    hash: "BEEFcafeaaaaAAAA...."
```

Comment:
You give what looks to be a case insensitive example here, but as far as I can see the actual comparison will be case sensitive. It might be nice to make this case insensitive, and also give an error if "hash" isn't an actual valid hash.

Other than that, this looks good to me, thanks!

Reply:
Ok, I pushed the update making the hash insensitive and also checking for invalids hashes.